inc/api.php

```
@@ -337,7 +337,57 @@ function djz_get_gamipress_user_data($request) {
     $next_level_points = $level * 100;
     $progress = min(100, (($points % 100) / 100) * 100);
 
-    // 4. Achievements
+    // 4. Tracks Downloaded
+    $total_tracks = 0;
```

PHPMD (2.15.0)

[error] The variable $total_tracks is not named in camelCase. (undefined)

(CamelCaseVariableName)